spec/unit/Ipv6Generator.spec.js

```diff
@@ -61,4 +69,76 @@ describe("IPV6 Generator", function() {
             expect(newConfig.config.ipv6).toEqual(`2001:0db8:85a3::${(i+1).toString(16)}`);
         }
     });
+
+
+    it("should ensure IPv6 addresses are only unique across one network", async function() {
```

Comment:
Potentially this needs an integration test against a real postgresql DB though.

Reply:
I resolved to port all these tests to the integration format and back them with real datastores.